worlds/musedash/Options.py

```diff
@@ -153,8 +153,9 @@ class TrapCountPercentage(Range):
     display_name = "Trap Percentage"
 
 
-class SongSet(OptionSet):
-    valid_keys = SONG_DATA.keys()
+class SongSet(OptionSet):    
+    valid_keys = {x for x in SONG_DATA.keys()}
```

Comment:
The preferred way to 'cast' an iterable to a set is by calling `set(iterable)` because, with CPython, it iterates in C code and has extra optimisations for common built-in types, whereas a set comprehension iterates and constructs the new set in Python code.
```suggestion
    valid_keys = set(SONG_DATA.keys())
```